refinery/ui/source/js/file-browser/ctrls/assay-filters-ctrl.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Assay Filters Ctrl
+ * @namespace AssayFiltersCtrl
```

Comment:
agreed....perhaps move it to common services or a separate module. It's really dependent on a specific api data structure. Let's wait until it works out for the user/files module and then discuss moving it.